app/controllers/application_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ApplicationController < FloodRiskEngine::ApplicationController
   include Pundit
 
+  layout ->(_) { "appplication" }
+
```

Comment:
I saw you adding some configuration to set the layout in other PR? so this was a place holder to pull that in ... i believe when it comes dynamically from a  variable you need this format but could be wrong ?